standalone-metastore/src/main/java/org/apache/hadoop/hive/metastore/ReplChangeManager.java

```diff
@@ -56,6 +58,7 @@
   private static final String ORIG_LOC_TAG = "user.original-loc";
   static final String REMAIN_IN_TRASH_TAG = "user.remain-in-trash";
   private static final String URI_FRAGMENT_SEPARATOR = "#";
+  public static final String SOURCE_OF_REPLICATION = "repl.source.for";
```

Comment:
Shall just use "repl.source" or "source_of_replication" instead of "repl.source.for".

Reply:
IDs are internal concept of Beacon. Hive has nothing to do with it. Even if they set anything, hive gonna work and some user uses it differently. So, it makes sense to keep a meaningful name for key as it is exposed.